src/components/CollapsibleFeedback/Feedback.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Collapsible, Icon } from '@edx/paragon';
+import { ExpandMore, ExpandLess } from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import messages from './messages';
+import InfoPopover from 'components/InfoPopover';
+
+const Feedback = ({
+  criterionName,
+  selectedOption,
+  selectedPoints,
+  commentHeader,
+  commentBody,
+  defaultOpen,
+}) => {
+  const [isExpanded, setIsExpanded] = React.useState(defaultOpen);
+  const { formatMessage } = useIntl();
+
+  return (
+    <>
+      <div className='mt-2'>
+        <div className='d-flex justify-content-between align-items-center'>
+          <h5 className='mb-0'>{criterionName}</h5>
+          <InfoPopover onClick={() => {}}>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam, quis nostrud exercitation ullamco laboris
+              nisi ut aliquip ex ea commodo consequat.
+            </p>
+          </InfoPopover>
+        </div>
+        {selectedOption && (
+          <p>
+            {selectedOption} -- {selectedPoints} points
+          </p>
+        )}
+      </div>
+      <div className='bg-gray-100 p-3'>
+        <Collapsible.Advanced
+          open={isExpanded}
+          onToggle={() => setIsExpanded(!isExpanded)}
```

Comment:
all of this logic needs to be pulled out into a hook, but for first pass this is fine.  next week when we are testing, we will need to be optimizing a bit